source/screens/caseScreens/CaseSummary.tsx

```diff
@@ -257,14 +259,12 @@ const computeCaseCardComponent = (
       onButtonClick={isClosed ? toggleModal : buttonProps.onClick}
       buttonIconName={isClosed ? "remove-red-eye" : "arrow-forward"}
       completions={unApprovedCompletionDescriptions}
+      completionsClarification={completionsClarification}
+      isRandomCheck={statusType.includes("randomCheck")}
```

Comment:
instead of having `isRandomCheck` as a prop, you can instead give the `completionsClarification` constant a value when it is a random check. like:

```js
const isRandomCheck = statusType.includes("randomCheck");
const completionsClarification = !isRandomCheck && caseItem?.details?.completions?.description || "";
``` 

And then you do not need to check for random check in the casecard component but only check for completionsClarification:

```js
{completionsClarification && (
  <CompletionsClarificationOutset>
    <Card.Text strong italic colorSchema="neutral">
      {completionsClarification}
    </Card.Text>
   </CompletionsClarificationOutset>
 )}

```

The casecard doesn't really need to know if its an random check or not